client/src/partials/vouchers/index.js

```diff
@@ -157,14 +157,15 @@ function VoucherController(Vouchers, $translate, Notify, Filtering, uiGridGroupi
     var dateFrom = vm.dateInterval ? vm.dateInterval.dateFrom: null;
     var dateTo = vm.dateInterval ? vm.dateInterval.dateTo : null;
     var url = '/vouchers/reports';
-    var params = { dateFrom: dateFrom, dateTo: dateTo, renderer: 'pdf' };
+    var params = { dateFrom: dateFrom, dateTo: dateTo, renderer: 'pdf', lang: Languages.key };
     Modal.openReports({ url: url, params: params });
   }
 
   // showReceipt
   function showReceipt(uuid) {
     var url = '/vouchers/receipts/' + uuid;
-    Modal.openReports({ url: url, renderer: 'pdf' });
+    var params = { renderer: 'pdf', lang: Languages.key };
```

Comment:
Cool!  In the future, we could probably do the same thing with `currency = ${id}` to allow reports to be rendered in multiple currencies.